prog-o-meter.py

```diff
@@ -195,8 +195,9 @@ def input_buttons(self):
         Displays a set of two radio buttons, for the user to indicate whether they are a new
         or returning user. Window closes when user clicks one of the buttons.
         """
-        Tk.Radiobutton(self.root, text = "I already have a meter", variable = self.choice, value = 1, command = self.close_window, indicatoron = 0).pack(pady = 5)
-        Tk.Radiobutton(self.root, text = "I don't have a meter yet", variable = self.choice, value = 2, command = self.close_window, indicatoron = 0).pack(pady = 5)
+        BTTN_WIDTH = 18 

```

Comment:
Just a small detail. For documentation purposes, please add a line after 196 that says

Attributes:
   BTTN_WIDTH: (explanation of the attribute here)

You can see an example of this in the docstring (comment) of the function canvas_layout(self) on line 180

Reply:
Alright I understand, I will add the documentation.

Reply:
The documentation has been added.